.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,54 @@
+name: Publish Docker image
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  checkchanged:
+    runs-on: ubuntu-latest
+    outputs:
+      changed: ${{ steps.step1.outputs.didChange }}
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - id: step1
+        run: >
+          chmod +x ./ops/ci-builder/check-changed.sh &&
```

Comment:
Might be able to remove the use of chmod here. Generally the ci runner will have full permissions. 

Optimism had to use this when constructing their docker container, but that’s a bit of a special case because they wanted the container to be able to use it while it’s running.